marklogic-data-hub/src/main/java/com/marklogic/hub/cli/InstallLocalDhfCommand.java

```diff
@@ -0,0 +1,117 @@
+package com.marklogic.hub.cli;
+
+import com.beust.jcommander.Parameters;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.appdeployer.ConfigDir;
+import com.marklogic.hub.impl.DataHubImpl;
+import com.marklogic.hub.impl.HubConfigImpl;
+import com.marklogic.mgmt.resource.security.UserManager;
+import com.marklogic.mgmt.util.ObjectMapperFactory;
+import com.marklogic.rest.util.ResourcesFragment;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+
+@Parameters(commandDescription = "Install or upgrade DHF into a local (non-DHS) environment")
+public class InstallLocalDhfCommand extends AbstractInstallerCommand {
+
+    private ObjectMapper objectMapper = ObjectMapperFactory.getObjectMapper();
+
+    private DataHubImpl dataHub;
+    private HubConfigImpl hubConfig;
```

Comment:
https://github.com/marklogic/marklogic-data-hub/blob/dae30aca47ce12b7bc934740ee1664d1acb73ae1/marklogic-data-hub/src/main/java/com/marklogic/hub/cli/InstallLocalDhfCommand.java#L25

could remove the line